pandas/tests/test_base.py

```diff
@@ -269,52 +269,50 @@ def setup_method(self, method):
         self.is_valid_objs = self.objs
         self.not_valid_objs = []
 
-    def test_none_comparison(self):
-
+    def test_none_comparison(self, series_fixture):
```

Comment:
so typically we will want to rename the fixture at the start of the method, e.g.
```
s = all_series_fixture
...
```